pom.xml

```diff
@@ -61,6 +61,12 @@
             <version>1.7.30</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.awaitility</groupId>
```

Comment:
Not sure the extra dependency is worth the functionality.

Reply:
Well, it is a test scoped dependency on purpose and awaitility has more features than a while loop. But this is just a suggestion of course.